apps/ubuntu/resources.qrc

```diff
@@ -0,0 +1,43 @@
+<RCC>
```

Comment:
What are the reasons/advantages of using resources? Is this something which should also be done for Sailfish OS?